IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -509,12 +558,15 @@ - (MSIDAccessToken *)getATForAccount:(MSIDAccount *)account
     }
     
     MSIDAccessToken *tokenToReturn = matchedTokens[0];
-    tokenToReturn.authority = parameters.authority;
+    
+    if (parameters.authority)
```

Comment:
this will now always be true if token is not nil, right?

Reply:
it could be false if there are more than one AT and parameters.authority was nil.